specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -82,7 +82,7 @@ EE platform or any profile on which it builds, as long as the
 corresponding compatibility requirements are satisfied.
```

Comment:
Missed a Java EE reference...

Reply:
Hi Kevin, Sorry I don't quite understand the changes that needs to be made here. Do I have to remove the "EE platform" ? Please I am quite a novice at this.

Thanks